be/src/runtime/user_function_cache.cpp

```diff
@@ -291,4 +296,29 @@ std::string UserFunctionCache::_make_lib_file(int64_t function_id, const std::st
     return fmt::format("{}/{}/{}.{}{}", _lib_dir, shard, function_id, checksum, shuffix);
 }
 
+Status UserFunctionCache::_remove_all_lib_file() {
+    // create library directory if not exist
+    RETURN_IF_ERROR(fs::create_directories(_lib_dir));
```

Comment:
Why need to create new directories when remove the lib files?

Reply:
> Why need to create new directories when remove the lib files?

After deleting the jar cache, the folder needs to still exist so that the runtime jar cache file can be written normally.

Reply:
So I think  we should check whether the dir exist, then we remove the files in the dirs.

A function should mainly do only one thing, and the implementation should be consistent with the naming.

Reply:
> So I think we should check whether the dir exist, then we remove the files in the dirs.
> 
> A function should mainly do only one thing, and the implementation should be consistent with the naming.

done